### ISSUE TITLE
[Support]: Connection refused after update?

### ISSUE THREAD

**Issue by bodleytunes** [sinamics/ztnet#587]
### 📝 Inquiry

Hi There,
I updated my instance yesterday with the usual docker compose pull && docker compose up -d, but this time even though it looks like its started and docker is listening on the correct port, I just get a connection refused when conecting to the web UI. 

I ran a tcp dump and can see TCP RST's straight away after the S / Syn packet.

It doesnt look like there are any errors in the main UI logs.

Is there anything else I can do to troubleshoot?

I even tried rolling back the docker instance / image tag and its still not working, which makes me dread that it might something database related.

Regards,
Jon.

### 🔖 Version

latest

### 🔧 Deployment Type

- [X] Docker
- [ ] Standalone

### 💻 Operating System

Ubuntu

### 🌐 Browser

Chrome

### 📚 Any Other Information That May Be Helpful

```
CONTAINER ID   IMAGE                       COMMAND                  CREATED        STATUS        PORTS                                       NAMES
916d0b72bb39   sinamics/ztnet:fallback     "/app/init-db.sh nod…"   21 hours ago   Up 21 hours   0.0.0.0:3000->3000/tcp, :::3000->3000/tcp   ztnet
2b2cd160dbf4   zyclonite/zerotier:1.14.0   "entrypoint.sh -U"       21 hours ago   Up 21 hours   0.0.0.0:9993->9993/udp, :::9993->9993/udp   zerotier
9e47048ffc9c   postgres:15.2-alpine        "docker-entrypoint.s…"   21 hours ago   Up 21 hours   5432/tcp
```

```
root@zt-controller:~/docker/zt-net# docker logs ztnet
Creating .env file...
Applying migrations to the database...
Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma
Datasource "db": PostgreSQL database "postgres", schema "public" at "postgres:5432"

36 migrations found in prisma/migrations


No pending migrations to apply.
npm notice
npm notice New minor version of npm available! 10.7.0 -> 10.9.1
npm notice Changelog: https://github.com/npm/cli/releases/tag/v10.9.1
npm notice To update run: npm install -g npm@10.9.1
npm notice
Migrations applied successfully!
Seeding the database...
Environment variables loaded from .env
Running seed command `ts-node --compiler-options {"module":"CommonJS"} prisma/seed.ts` ...
Seeding:: User Options complete!
Seeding:: Updating user ID complete!

🌱  The seed command has been executed.
┌─────────────────────────────────────────────────────────┐
│  Update available 5.19.1 -> 5.22.0                      │
│  Run the following to update                            │
│    npm i -g prisma@latest                               │
│    npm i @prisma/client@latest                          │
└─────────────────────────────────────────────────────────┘
Database seeded successfully!
Executing command
   ▲ Next.js 14.1.4
   - Local:        http://916d0b72bb39:3000
   - Network:      http://[fd12:3456:789a::4]:3000

 ✓ Ready in 150ms
Creating .env file...
Applying migrations to the database...
Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma
Datasource "db": PostgreSQL database "postgres", schema "public" at "postgres:5432"

36 migrations found in prisma/migrations


No pending migrations to apply.
Migrations applied successfully!
Seeding the database...
Environment variables loaded from .env
Running seed command `ts-node --compiler-options {"module":"CommonJS"} prisma/seed.ts` ...
Seeding:: User Options complete!
Seeding:: Updating user ID complete!

🌱  The seed command has been executed.
Database seeded successfully!
Executing command
   ▲ Next.js 14.1.4
   - Local:        http://916d0b72bb39:3000
   - Network:      http://[fd12:3456:789a::4]:3000

 ✓ Ready in 145m```

```

Postgres
```
PostgreSQL Database directory appears to contain a database; Skipping initialization

2024-11-27 02:46:14.856 UTC [1] LOG:  starting PostgreSQL 15.2 on x86_64-pc-linux-musl, compiled by gcc (Alpine 12.2.1_git20220924-r4) 12.2.1 20220924, 64-bit
2024-11-27 02:46:14.856 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2024-11-27 02:46:14.856 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2024-11-27 02:46:14.858 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2024-11-27 02:46:14.864 UTC [24] LOG:  database system was shut down at 2024-11-27 02:45:52 UTC
2024-11-27 02:46:14.870 UTC [1] LOG:  database system is ready to accept connections
2024-11-27 02:46:16.150 UTC [30] LOG:  could not receive data from client: Connection reset by peer
2024-11-27 02:48:29.608 UTC [1] LOG:  received fast shutdown request
2024-11-27 02:48:29.609 UTC [1] LOG:  aborting any active transactions
2024-11-27 02:48:29.612 UTC [1] LOG:  background worker "logical replication launcher" (PID 27) exited with exit code 1
2024-11-27 02:48:29.613 UTC [22] LOG:  shutting down
2024-11-27 02:48:29.615 UTC [22] LOG:  checkpoint starting: shutdown immediate
2024-11-27 02:48:29.622 UTC [22] LOG:  checkpoint complete: wrote 2 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.003 s, sync=0.001 s, total=0.009 s; sync files=3, longest=0.001 s, average=0.001 s; distance=0 kB, estimate=0 kB
2024-11-27 02:48:29.626 UTC [1] LOG:  database system is shut down

PostgreSQL Database directory appears to contain a database; Skipping initialization

2024-11-27 02:48:45.330 UTC [1] LOG:  starting PostgreSQL 15.2 on x86_64-pc-linux-musl, compiled by gcc (Alpine 12.2.1_git20220924-r4) 12.2.1 20220924, 64-bit
2024-11-27 02:48:45.330 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2024-11-27 02:48:45.330 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2024-11-27 02:48:45.333 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2024-11-27 02:48:45.338 UTC [24] LOG:  database system was shut down at 2024-11-27 02:48:29 UTC
2024-11-27 02:48:45.344 UTC [1] LOG:  database system is ready to accept connections
2024-11-27 02:48:46.528 UTC [29] LOG:  could not receive data from client: Connection reset by peer
2024-11-27 02:53:45.434 UTC [22] LOG:  checkpoint starting: time
2024-11-27 02:53:45.446 UTC [22] LOG:  checkpoint complete: wrote 2 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.004 s, sync=0.002 s, total=0.013 s; sync files=3, longest=0.001 s, average=0.001 s; distance=0 kB, estimate=0 kB
```

_No response_

**Comment by sinamics** [comment]
I think your issue is ipv6 related.
`Network:      http://[fd12:3456:789a::4]:3000`

can you post your docker-compose file, or test the default config:
https://ztnet.network/installation/docker-compose#setup

**Comment by bodleytunes** [comment]
Yeah it was ipv6 I'd turned it on the docker bridge messing around.

Thanks for the heads up!

**Comment by gomaaz** [comment]
I have this error this morning. I'm not using any ipv6. 

they are on a same docker-network.

Any suggestions?

ZTnet docker
```
Creating .env file...
Applying migrations to the database...
Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma
Datasource "db": PostgreSQL database "ztnet", schema "public" at "postgres:5432"
39 migrations found in prisma/migrations
No pending migrations to apply.
npm notice
npm notice New major version of npm available! 10.7.0 -> 11.4.2
npm notice Changelog: https://github.com/npm/cli/releases/tag/v11.4.2
npm notice To update run: npm install -g npm@11.4.2
npm notice
Migrations applied successfully!
Seeding the database...
Environment variables loaded from .env
Running seed command `ts-node --compiler-options {"module":"CommonJS"} prisma/seed.ts` ...
Seeding:: User Options complete!
Seeding:: Updating user ID complete!
🌱  The seed command has been executed.
┌─────────────────────────────────────────────────────────┐
│  Update available 6.9.0 -> 6.11.1                       │
│  Run the following to update                            │
│    npm i -g prisma@latest                               │
│    npm i @prisma/client@latest                          │
└─────────────────────────────────────────────────────────┘
Database seeded successfully!
Executing command
   ▲ Next.js 15.2.4
   - Local:        http://198fcd8cbe8c:3000
   - Network:      http://198fcd8cbe8c:3000
 ✓ Starting...
 ✓ Ready in 252ms
```


Postgres docker
```
PostgreSQL Database directory appears to contain a database; Skipping initialization
2025-07-04 12:05:03.822 UTC [1] LOG:  starting PostgreSQL 15.2 on x86_64-pc-linux-musl, compiled by gcc (Alpine 12.2.1_git20220924-r4) 12.2.1 20220924, 64-bit
2025-07-04 12:05:03.822 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2025-07-04 12:05:03.822 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2025-07-04 12:05:03.824 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-07-04 12:05:03.828 UTC [24] LOG:  database system was interrupted; last known up at 2025-07-04 09:45:59 UTC
2025-07-04 12:05:03.948 UTC [24] LOG:  database system was not properly shut down; automatic recovery in progress
2025-07-04 12:05:03.951 UTC [24] LOG:  redo starts at 0/18AC4D30
2025-07-04 12:05:03.953 UTC [24] LOG:  invalid record length at 0/18ACD310: wanted 24, got 0
2025-07-04 12:05:03.953 UTC [24] LOG:  redo done at 0/18ACD2D8 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-07-04 12:05:03.957 UTC [22] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-07-04 12:05:03.963 UTC [22] LOG:  checkpoint complete: wrote 10 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.002 s, sync=0.003 s, total=0.008 s; sync files=7, longest=0.001 s, average=0.001 s; distance=33 kB, estimate=33 kB
2025-07-04 12:05:03.967 UTC [1] LOG:  database system is ready to accept connections
2025-07-04 12:05:05.224 UTC [29] LOG:  could not receive data from client: Connection reset by peer
2025-07-04 12:05:05.328 UTC [30] LOG:  could not receive data from client: Connection reset by peer
2025-07-04 12:05:15.882 UTC [34] LOG:  could not receive data from client: Connection reset by peer


```

**Comment by gomaaz** [comment]
seems like chrome has a problem with this site....
I can access everything on the webinterface by using firefox....weird.
